### PR TITLE
add plan column to runtime. output also make it a filtering flag

### PR DESCRIPTION
**Pull request by life0215**
…runtime cmd flag

**Description**

Changes proposed in this pull request:

- add coloumn PLAN to KCP CLI runtime cmd output 
- add plan as KCP CLI runtime cmd flag

**Related issue(s)**

### CODE REVIEW

**Comment by life0215** [comment]
This PR has a dependency on below PR to unblock `pre-master-kcp-cli`  job
https://github.com/kyma-project/control-plane/pull/394

File: tools/cli/pkg/command/runtime.go [hunk 1]
```diff
@@ -72,6 +76,7 @@ The command supports filtering Runtimes based on various attributes. See the lis
 	cobraCmd.Flags().StringSliceVarP(&cmd.params.SubAccountIDs, "subaccount", "s", nil, "Filter by subaccount ID. You can provide multiple values, either separated by a comma (e.g. SAID1,SAID2), or by specifying the option multiple times.")
 	cobraCmd.Flags().StringSliceVarP(&cmd.params.RuntimeIDs, "runtime-id", "i", nil, "Filter by Runtime ID. You can provide multiple values, either separated by a comma (e.g. ID1,ID2), or by specifying the option multiple times.")
 	cobraCmd.Flags().StringSliceVarP(&cmd.params.Regions, "region", "r", nil, "Filter by provider region. You can provide multiple values, either separated by a comma (e.g. westeurope,northeurope), or by specifying the option multiple times.")
+	cobraCmd.Flags().StringSliceVarP(&cmd.params.Plans, "plan", "p", nil, "Filter by service plan name. You can provide multiple values, either separated by a comma (e.g. azure,trial), or by specifying the option multiple times.")
```
Comment (ebensom) [inline tools/cli/pkg/command/runtime.go:79]: You need to run `make docs` to regenerate the Markdown documents.

**Comment by life0215** [comment]
[It's](https://status.build.kyma-project.io/view/gcs/kyma-prow-logs/pr-logs/pull/kyma-project_control-plane/396/pre-master-kcp-cli/1339721128301563904)  downloading old version of control-plane, interesting...
`go: downloading github.com/kyma-project/control-plane v0.0.0-20201211152036-9bdabffd55fb`

File: tools/cli/pkg/command/runtime.go (hunk 1 above)
Comment (life0215) [reply tools/cli/pkg/command/runtime.go:79]: thanks, docs added

**Review by ebensom — APPROVED** [review]

**Comment by life0215** [comment]
@klaudiagrz  Please help review when you are back :)

**Review by klaudiagrz — APPROVED** [review]

**Pull request merged**